test/parser.spec.js

```diff
@@ -41,9 +41,9 @@ describe('Parser', function() {
       '\n# tar' +
       '\n> archiving utility' +
       '\n' +
-      '\n- create an archive' +
+      '\ncreate an archive' +
```

Comment:
Sure, I understand that it doesn't care, but since this code is meant to represent real tldr-pages content, it would make sense IMO to match the actual format, for consistency. But of course, that change is not directly related to this PR, so it shouldn't hold up merging.